config/routes.rb

```diff
@@ -19,19 +19,13 @@
     get  'import_progress'
   end
 
-  resources :tag_migrations, only: %i(index show create) do
+  resources :tag_migrations, only: [:index, :new, :create, :show, :destroy] do
```

Comment:
What do you think of writing `except: %i(edit update)` instead (as we are allowing most actions)?